cpu/cpustress.py

```diff
@@ -267,7 +266,7 @@ def dlpar_cpu_hotplug(self):
         PowerVM and Guest only: Dynamic Resource Manager
         use drmgr command to hotplug and hotunplug cpus
         """
-        if 'PowerNV' not in open('/proc/cpuinfo', 'r').read():
+        if 'PowerNV' not in cpu._get_cpu_info():
```

Comment:
Function `_get_cpu_info` does not return with the `PowerNV` tag. Only `Returns info on the 1st CPU entry from /proc/cpuinfo as a list of lines` which does not contain `PowerNV` tag, and hence cannot be used for this purpose.

**Cpuinfo**
```
processor	: 0
cpu		: POWER9, altivec supported
clock		: 3783.000000MHz
revision	: 2.2 (pvr 004e 1202)

processor	: 1
cpu		: POWER9, altivec supported
clock		: 3783.000000MHz
revision	: 2.2 (pvr 004e 1202)
...
...
timebase	: 512000000
platform	: PowerNV
model		: 0000000000000000
machine		: PowerNV 0000000000000000
firmware	: OPAL
MMU		: Radix
```
What is being returned.
```
>>> cpu._get_cpu_info()
['processor\t: 0\n', 'cpu\t\t: POWER9, altivec supported\n', 'clock\t\t: 3783.000000MHz\n', 'revision\t: 2.2 (pvr 004e 1202)\n']
```